mesonbuild/msubprojects.py

```diff
@@ -257,9 +257,8 @@ def foreach(wrap, repo_dir, options):
 def add_common_arguments(p):
     p.add_argument('--sourcedir', default='.',
                    help='Path to source directory')
-    p.add_argument('--type', default='',
-                   choices=['file', 'git', 'hg', 'svn'],
-                   help='Only subprojects of given type (default: all)')
+    p.add_argument('--types', default='',
+                   help='Only subprojects of given types, comma separated (default: all)')
```

Comment:
Sorry to nitpick, but we should list the types that are available. `file` for example is not discoverable :)

Reply:
True, fixed now.